### PR TITLE
app-template

### PR DESCRIPTION
**Pull request by diesttro**

### CODE REVIEW

File: staff/carlos-diestro/app-template/news.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      <li><a href="./profile.html">profile</a></li>
+      <li><a href="./about.html">about</a></li>
+      <li><a href="./contact.html">contact</a></li>
+      <li><a href="./login.html">login</a></li>
+      <li><a href="./register.html">register</a></li>
+    </ul>
+  </nav>
+  <main>
+    <article>
+      <h3>Lorem ipsum, dolor sit amet consectetur adipisicing elit</h3>
```
Comment (almuitur) [inline staff/carlos-diestro/app-template/news.html]: Why not using a header for news?

File: staff/carlos-diestro/app-template/profile.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      <li><a href="./about.html">about</a></li>
+      <li><a href="./contact.html">contact</a></li>
+      <li><a href="./login.html">login</a></li>
+      <li><a href="./register.html">register</a></li>
+    </ul>
+  </nav>
+  <main>
+    <h2>Your profile</h2>
+    <section>
+      <b>Name:</b> Carlos
```
Comment (almuitur) [inline staff/carlos-diestro/app-template/profile.html:27]: Why not using address?

File: staff/carlos-diestro/app-template/about.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      <li><a href="./home.html">home</a></li>
+      <li><a href="./news.html">news</a></li>
+      <li><a href="./profile.html">profile</a></li>
+      <li><a href="./about.html">about</a></li>
+      <li><a href="./contact.html">contact</a></li>
+      <li><a href="./login.html">login</a></li>
+      <li><a href="./register.html">register</a></li>
+    </ul>
+  </nav>
+  <main>
```
Comment (maryammdot) [inline staff/carlos-diestro/app-template/about.html:24]: i don't see where do you close `<main>`

File: staff/carlos-diestro/app-template/news.html (hunk 1 above)
Comment (diesttro) [reply staff/carlos-diestro/app-template/news.html]: I had not thought of it, I'll try it

File: staff/carlos-diestro/app-template/about.html (hunk 3 above)
Comment (diesttro) [reply staff/carlos-diestro/app-template/about.html:24]: I don't see it either, I'll fix it

File: staff/carlos-diestro/app-template/profile.html (hunk 2 above)
Comment (diesttro) [reply staff/carlos-diestro/app-template/profile.html:27]: It's only a dumb text, but I'm not sure that addres tag will be ok here

Comment (Sergioluz90) [reply staff/carlos-diestro/app-template/profile.html:27]: I think @almuitur is right. 
"The address tag defines the contact information for the author/owner of a document or an article."
https://www.w3schools.com/tags/tag_address.asp

Comment (diesttro) [reply staff/carlos-diestro/app-template/profile.html:27]: Done

**Pull request merged**